src/getScores.js

```diff
@@ -0,0 +1,34 @@
+const getScores = async () => {
+  let leaderboardScores;
+  const scoreListContainer = document.getElementById('scoreList');
+  scoreListContainer.innerHTML = '';
+
+  try {
+    const leaderboardAPIrequest = await fetch(
+      'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/afMNT2NsfBzm4rDRXx1A/scores/',
+      {
+        method: 'GET',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      },
+    );
+
```

Comment:
The app does not function as expected. The user is unable to submit data and the score list does not change on clicking the refresh button. Please make sure the app is working as expected.

Reply:
What is the message you get, In my its working perfectly

PS: inputs cannot be empty